tests/testutils.py

```diff
@@ -1,7 +1,33 @@
-from django.test import SimpleTestCase
+from unittest.mock import Mock
+
+from django.template import Template, Context
+from django.template.response import TemplateResponse
+from django.test import SimpleTestCase, TestCase
+
+from django_components.middleware import ComponentDependencyMiddleware
+
+# Create middleware instance.  get_response function is not used, so pass a do-nothing lambda
+middleware = ComponentDependencyMiddleware(get_response=lambda _: None)
 
 
 class Django30CompatibleSimpleTestCase(SimpleTestCase):
     def assertHTMLEqual(self, left, right):
         left = left.replace(' type="text/javascript"', '')
         super(Django30CompatibleSimpleTestCase, self).assertHTMLEqual(left, right)
+
+
+class Django30CompatibleTestCase(TestCase):
```

Comment:
This can inherit from Django30CompatibleSimpleTestCase to avoid duplicated code.

Reply:
Done.